checkbox-provider-ce-oem/units/cpu/jobs.pxu

```diff
@@ -6,7 +6,9 @@ plugin: resource
 user: root
 category_id: com.canonical.plainbox::cpu
 estimated_duration: 1s
-command: cpufreq_governors.py --policy-resource
+command: 
+    cpufreq_governors.py --probe-module
```

Comment:
I think we should have another job to probe the kernel module, because of the `cpufreq_policy_list` job is a resource job, so we should prevent any unexpected output in it.